### ISSUE TITLE
Breaks CSSTransitionGroup enter animations

### ISSUE THREAD

**Issue by anthonator** [JedWatson/react-select#165]
When using react-select I'm seeing `enter` animations breaking. They simply don't animate. The `leave` animations continue to work. If I remove the `Select` component from my view the animations start working again.

**Comment by VictorGarcia** [comment]
Any updates on this? I'm experiencing the same issue...

**Comment by B-Reif** [comment]
I'm having this issue too (almost a year later). I was able to restore the animation by setting `searchable={false}`.

Maybe the search input box is slowing down render times too much? Maybe this is related somehow to [#733](https://github.com/JedWatson/react-select/issues/733).

**Comment by mmellado** [comment]
Bump, experiencing this as well

**Comment by jjmr** [comment]
still happening after 3 years...any clue?

**Comment by jossmac** [comment]
Is this still an issue in v2?

**Comment by jossmac** [comment]
Could someone please provide a reproduction in [CodeSandbox](http://codesandbox.io)?

**Comment by bryant-st** [comment]
same issue here

**Comment by foxinni** [comment]
It seems like `isSearchable={false}` for V2 fixes the CSS transition.

**Comment by OwenFlood** [comment]
I set up a quick codesandbox with both the searchable enabled and disabled to show the animation working/not working.

https://codesandbox.io/s/affectionate-rhodes-qir59

**Comment by twheys** [comment]
@gwyneplaine Could you please reopen this ticket given that @OwenFlood has posted a codesandbox link? This bug is still a thing.

**Comment by saltnpixels** [comment]
I made a useEffect that change the isSearchable to true after 1 millisecond....

```
const [isSearchable, setIsSearchable] = useState(false)
setTimeout(() => {
         setIsSearchable(true)
      }, 1)
```

But ultimately this needs a real fix....

**Comment by ebonow** [comment]
> I set up a quick codesandbox with both the searchable enabled and disabled to show the animation working/not working.
> 
> https://codesandbox.io/s/affectionate-rhodes-qir59

I tested the link as provided, confirmed that this is indeed still an issue, and will reopen this.

@marianayap has been actively volunteering for issues and if she or anyone else would like to work on a fix and PR, then we can add this to the queue of PRs for release.

As an update to everyone, you can [read more here](https://github.com/JedWatson/react-select/issues/4293), but... 
TLDR: We've added contributors and currently choosing PR candidates for v3.2.

**Comment by ebonow** [comment]
As expected, this appears to be an issue related to the `AutosizeInput` used in the Input component.

Here is a fork demonstrating the same behavior of AutosizeInput vs input [here](https://codesandbox.io/s/react-select-css-transition-wvixr?file=/src/index.js)
